src/App.tsx

```diff
@@ -20,7 +20,9 @@ export const App = () => (
             <Route index element={<HomePage />} />
             <Route path="home" element={<Navigate to="/" replace />} />
             <Route path="phones" element={<PhonesPage />} />
-            <Route path="item-details/1" element={<ItemDetails />} />
+            <Route path="item-details">
+              <Route path=":id" element={<ItemDetails />} />
+            </Route>
```

Comment:
I'll need you to make a pull from main, beacause this route was already implemented. And merging this will break other's code.